app/javascript/packs/scroll.js

```diff
@@ -0,0 +1,12 @@
+window.addEventListener('scroll', () => {
```

Comment:
why the window object?

Reply:
i think it's appropriate to use it for those purposes